### DISCUSSION TITLE
Define global fonts, such as family and size

### DISCUSSION THREAD

**Discussion by funchan24** [israel-dryer/ttkbootstrap#206] [Q&A] [answered]
Define global fonts, such as family and size #205 
example by @israel-dryer  

import ttkbootstrap as ttk
from ttkbootstrap.constants import *
from ttkbootstrap import font

app = ttk.Window()

default_font = font.nametofont('TkDefaultFont')
default_font['size'] = 16

lbl = ttk.Label(text="Testing")
lbl.pack(padx=20, pady=20)

app.mainloop()

**Comment by israel-dryer** [comment, marked as answer]
global font options can be set as in 
```python
import ttkbootstrap as ttk
from ttkbootstrap.constants import *
from ttkbootstrap import font

app = ttk.Window()

default_font = font.nametofont('TkDefaultFont')
default_font['size'] = 16

lbl = ttk.Label(text="Testing")
lbl.pack(padx=20, pady=20)

app.mainloop()
```
Regular font
![python_C4qCWA5FKp](https://user-images.githubusercontent.com/42584742/153992751-70543e0d-c258-4c15-a33b-2ad96d363958.png)

16pt font
![python_9ZlCg8odMv](https://user-images.githubusercontent.com/42584742/153992649-bc77e3c6-cbf1-45f5-be18-c57ff5e06a86.png)

**Reply by 44762339** [reply]
how to do partly options such as in buttons

**Reply by israel-dryer** [reply]
Are you asking how to change the font for buttons?

**Reply by 44762339** [reply]
yes,thanks.
